articles/manual/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Q# Manual | Microsoft Docs 
+description: Q# Manual
+author: E. Gonzalez
+ms.author: v-edsanc@microsoft.com 
+ms.date: 07/31/2019
+ms.topic: article
+uid: microsoft.quantum.manual.intro
+---
+
+# Q# Manual
+
+## What is Q#?
+
+Q# (Q-sharp) is a domain-specific programming language used for
+expressing quantum algorithms.
+It is to be used for writing subroutines that execute on an adjunct
+quantum processor, under the control of a classical host program and computer.
+Until quantum processors are widely available, Q# subroutines execute on a simulator.
+
+A natural model for quantum computation is to treat the quantum computer
+as a coprocessor, similar to that used for GPUs, FPGAs, and other adjunct
+processors.
+The primary control logic runs classical code on a classical "host" computer.
+When appropriate and necessary, the host program can invoke a subroutine
+that runs on the adjunct processor.
+When the subroutine completes, the host program gets access to the
+subroutine's results.
+
+Q# provides a small set of primitive types, along with two ways
+(arrays and tuples) for creating new, structured types.
+It supports a basic procedural model for writing programs,
+with loops and if/then statements.
+The top-level constructs in Q# are user defined types, operations,
+and functions.
+
+Q# is scalable: it can be used to write simple demonstration programs like teleport that run on a few qubits, but also supports writing large, sophisticated programs such as simulations of complex molecules that will require large machines with millions of qubits. Even though large physical machines are still in the future, Q# allows a programmer to program complex quantum algorithms now. What is more, Q# supports various tasks such as debugging, profiling, resource estimation, and certain special-purpose simulations in a scalable way.
+
+## Why Q#?
+
+[![Quantum computing video](http://img.youtube.com/vi/cOUrzxyng04/0.jpg)](http://www.youtube.com/watch?v=cOUrzxyng04 "")
```

Comment:
This video isn't showing up as a video for some reason, such that it isn't obvious that I should be able to click through and watch it. It may also be good to check if there's a more up-to-date video that we can use; @bromeg may have some resources to suggest there.

Reply:
I solved the issue in cd720b60367aedac7bba5f431034ff0881c41c2e , now it shows up correctly. 
And yes, I agree, the video is not completely appropriate, it's quite general about Quantum Computing and barely mentions anything related specifically with Q#, but I couldn't find anything better.